func-consumption/package.json

```diff
@@ -7,7 +7,7 @@
     "node": ">= 10.14.1"
```

Comment:
have you tried manually removing the `devDependencies` from this file to see if they are actually needed and/or re-added?

Reply:
Not as such.

The gulp stuff we clearly need, and I *believe* that IntelliJ in-editor linting *requires* Eslint et al. in order to work for monorepos.

Are there any packages you think could be ditched?

Reply:
the jest stuff should not be in there, as the app itself does no testing.  
it may be a case of pruning each one in turn, to figure out what is needed.

there is some other cruft that we deploy which i noticed today, which we should discuss elsewhere.